flask_common/crypto.py

```diff
@@ -32,41 +43,52 @@ def aes_generate_key():
 # Encrypt + sign using a random IV
 def aes_encrypt(key, data):
     assert len(key) == KEY_LENGTH, 'invalid key size'
-    iv = rng(AES_BLOCK_SIZE)
-    return iv + aes_encrypt_iv(key, data, iv)
+    iv = rng(IV_SIZE)
+    return V1_MARKER + iv + aes_encrypt_iv(key, data, iv)
 
 
 # Verify + decrypt data encrypted with IV
 def aes_decrypt(key, data):
     assert len(key) == KEY_LENGTH, 'invalid key size'
-    iv = data[:AES_BLOCK_SIZE]
-    data = data[AES_BLOCK_SIZE:]
-    return aes_decrypt_iv(key, data, iv)
+    is_v0 = False
```

Comment:
I don't think we should have `is_v?` variables. We should just extract the version and pass it to all functions so they can act on it as they see fit.